rust/user-info-fetcher/src/main.rs

```diff
@@ -205,6 +209,11 @@ struct UserInfo {
 enum GetUserInfoError {
     #[snafu(display("failed to get user information from Keycloak"))]
     Keycloak { source: backend::keycloak::Error },
+
+    #[snafu(display(
+        "failed to get user information from the Authentication & Authorization Service"
+    ))]
+    Aas { source: backend::aas::Error },
```

Comment:
Also here, AAS is waaaaaay too generic to be used without a qualifier.

Reply:
https://github.com/stackabletech/opa-operator/pull/519/commits/6f0e8cd0b79752fd1cf3b75a9a36210b343dd27b